src/views/AddItem.jsx

```diff
@@ -16,14 +16,57 @@ export function AddItem({ listToken }) {
 	const handleSubmit = async (e) => {
 		e.preventDefault();
 
+		// const isValid = (itemName) => {
+		// 	checkEmpty(itemName);
+		// 	checkDuplicates(itemName);
+		// }
```

Comment:
Here again you need to return a boolean, because it's like you're asking "Is an item name valid?". An item name is valid when it's not empty and there aren't any duplicates. So you need to return the boolean result of this condition.